### DISCUSSION TITLE
I keep getting----- Error: The YouTube API key you provided was not a string.

### DISCUSSION THREAD

**Discussion by idkcoderman** [eritislami/evobot#580] [Q&A]
I have it saying this

"YOUTUBE_API_KEY": _____________________

(the underline is pretending to be my key)

i need to know how to fix this so I can run the bot.

**Comment by Takiyo0** [comment]
is it on your config.json? if yes, it must be ``` "YOUTUBE_API_KEY: "YOUR_KEY_HERE" ``` and not ``` "YOUTUBE_API_KEY": YOUR_KEY_HERE ``` 
based on how you type here ```"YOUTUBE_API_KEY": _____________________```
